src/widgets/peoplePicker.js

```diff
@@ -139,30 +139,19 @@ export class PeoplePicker {
     })
   }
 
-  createNewGroup (book) {
+  createNewGroup (book, defaultNewGroupName) {
     const { groupIndex, groupContainer } = indexes(book)
     const group = rdf.sym(
       `${groupContainer.uri}${uuid.v4().slice(0, 8)}.ttl#this`
     )
-    const name = this.options.defaultNewGroupName || 'Untitled Group'
```

Comment:
I could put this back, I have left it for your review because I think it's a change we would want to make later anyway.

Reply:
I agree